src/patternfly/components/Button/styles.scss

```diff
@@ -408,4 +411,8 @@
       border: var(--pf-c-button--BorderWidth) solid var(--pf-c-button--m-disabled--BorderColor);
     }
   }
+
+  &.pf-m-lg {
```

Comment:
@kybaker can you please take a look at this. If the icons on the header toolbar weren't designed to be larger than normal buttons, then we need to change the btn instead of creating a new modifier to change ths size.

Reply:
Updated